examples/model_hosting/notebook/training/LocalTraining.ipynb

```diff
@@ -295,7 +295,7 @@
     "df[\"predicted_friction\"] = predictions\n",
     "\n",
     "%matplotlib inline\n",
-    "df.plot(x=\"timestamp\")"
+    "df.plot()"
```

Comment:
Don't need/want `x="timestamp"` anymore?

Reply:
As above, not anymore.